.github/workflows/CI-unixish.yml

```diff
@@ -560,7 +560,7 @@ jobs:
           mkdir b2
           ./cppcheck $selfcheck_options $cppcheck_options --cppcheck-build-dir=b2 -DQT_VERSION=0x060000 -DQ_MOC_OUTPUT_REVISION=68 -DQT_CHARTS_LIB --library=qt --addon=naming.json -Icmake.output/gui -Igui gui/*.cpp cmake.output/gui || ec=1
           # self check test and tools
-          ./cppcheck $selfcheck_options $cppcheck_options -Icli test/*.cpp tools/*.cpp || ec=1
+          ./cppcheck $selfcheck_options $cppcheck_options -Icli test/*.cpp tools/dmake/*.cpp || ec=1
```

Comment:
you can remove the `*.cpp` in the dmake folder.

Reply:
I wonder if we can self check the tools/triage folder also.
And would it make sense to selfcheck the tools/defines folder? I am not sure how that is used off the top of my head.

Reply:
`triage` is checked in a different invocation since it required different options.

I forgot about `tools/defines`. I will have a look.

Reply:
`tools/defines` is not yet warnings-free (and still WIP - getting too distracted by other things) so I would like to defer that for now (also needs a test). I am also not sure if that would actually make sense.